pi-manage

```diff
@@ -586,6 +586,25 @@ def create(name, scope, action):
     return r
 
 
+@policy_manager.command
+def create_from_file(name, filename):
```

Comment:
Please consider naming the function "create". So the command line will be

     pi-manage policy create --filename pol.json

The "create" command would be the same like for creating realms or resolvers.